<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS>
<TS version="2.1" language="de_DE">
<context>
    <name>Core::Json</name>
    <message>
        <location filename="../source/FileHelper.cpp" line="65"/>
        <source>Could not open %1 for reading.</source>
        <translation>%1 kann icht zum lesen geöffnet werden.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.cpp" line="79"/>
        <source>Could not open %1 for writing.</source>
        <translation>%1 kann icht zum schreiben geöffnet werden.</translation>
    </message>
</context>
<context>
    <name>CxApp</name>
    <message>
        <location filename="../source/CxApp.cpp" line="31"/>
        <source>Cannot create log file directory: &apos;%1&apos;.</source>
        <translation>Logdateiverzeichnis &apos;%1&apos; konnte nicht angelegt werden.</translation>
    </message>
    <message>
        <location filename="../source/CxApp.cpp" line="32"/>
        <source>Cannot open log file: &apos;%1&apos;.</source>
        <translation>Logdatei &apos;%1&apos; konnte nicht geöffnet werden.</translation>
    </message>
    <message>
        <location filename="../source/CxApp.cpp" line="33"/>
        <source>%1 is running already!</source>
        <translation>%1 lauft bereits!</translation>
    </message>
    <message>
        <location filename="../source/CxApp.cpp" line="35"/>
        <source>Invalid parameter. Argument ignored: &apos;%1&apos;</source>
        <translation>Ungültiger Parameter. Argument übersprungen: &apos;%1&apos;</translation>
    </message>
    <message>
        <location filename="../source/CxApp.cpp" line="203"/>
        <location filename="../source/CxApp.cpp" line="223"/>
        <source>%1 command line help</source>
        <translation>%1 Kommandozeilenhilfe</translation>
    </message>
    <message>
        <location filename="../source/CxApp.cpp" line="209"/>
        <source>&lt;TABLE WIDTH=700 BORDER=1 CELLPADDING=7 CELLSPACING=0&gt;&lt;TR&gt;&lt;TH COLSPAN=2&gt;&lt;h2&gt;%1 command line parameters&lt;/h2&gt;&lt;/TH&gt;&lt;/TR&gt;</source>
        <translation>&lt;TABLE WIDTH=700 BORDER=1 CELLPADDING=7 CELLSPACING=0&gt;&lt;TR&gt;&lt;TH COLSPAN=2&gt;&lt;h2&gt;%1 Kommandozeilenparameter&lt;/h2&gt;&lt;/TH&gt;&lt;/TR&gt;</translation>
    </message>
    <message>
        <location filename="../source/CxApp.cpp" line="241"/>
        <source>&lt;h1&gt;Command line parameter errors found:&lt;/h1&gt;</source>
        <translation>&lt;h1&gt;Kommandozeilenparameterfehler gefunden:&lt;/h1&gt;</translation>
    </message>
    <message>
        <location filename="../source/CxApp.cpp" line="251"/>
        <source>%1 command line errors</source>
        <translation>%1 Kommandozeilenparameterfehler</translation>
    </message>
</context>
<context>
    <name>DatabaseImage</name>
    <message>
        <location filename="../source/Pixmap.h" line="65"/>
        <source>Images of Macro</source>
        <translation>Makro-Bilder</translation>
    </message>
    <message>
        <location filename="../source/Pixmap.h" line="66"/>
        <source>Image of Macro</source>
        <translation>Makro-Bild</translation>
    </message>
    <message>
        <location filename="../source/Pixmap.h" line="67"/>
        <source>Foreign Id</source>
        <translation>Externer Schlüssel</translation>
    </message>
    <message>
        <location filename="../source/Pixmap.h" line="68"/>
        <source>Description of the picture</source>
        <translation>Beschreibung des Bilds</translation>
    </message>
</context>
<context>
    <name>FileTool</name>
    <message>
        <location filename="../source/FileTool.h" line="31"/>
        <source>This will be skipped.</source>
        <translation>Wird übersprungen</translation>
    </message>
    <message>
        <location filename="../source/FileTool.h" line="32"/>
        <source>The operation will be canceled.</source>
        <translation>Vorgang wird abgebrochen.</translation>
    </message>
    <message>
        <location filename="../source/FileTool.cpp" line="46"/>
        <source>File %1 does not exist.</source>
        <translation>Daei %1 existiert nicht.</translation>
    </message>
    <message>
        <location filename="../source/FileTool.cpp" line="62"/>
        <source>1 file could not be found:
</source>
        <translation>Eine Datei wurde nicht gefunden:
</translation>
    </message>
    <message>
        <location filename="../source/FileTool.cpp" line="64"/>
        <source>%1 files could not be found:
</source>
        <translation>%1 Dateien wurden nicht gefunden:
</translation>
    </message>
    <message>
        <location filename="../source/FileTool.cpp" line="74"/>
        <source>[%1 other files]</source>
        <translation>[%1 andere Dateien]</translation>
    </message>
    <message>
        <location filename="../source/FileTool.cpp" line="103"/>
        <source>Directory %1 does not exist.</source>
        <translation>Verzeichnis %1 existiert nicht.</translation>
    </message>
    <message>
        <location filename="../source/FileTool.cpp" line="115"/>
        <source>Could not create subdirectory in %1. Please ensure that you have read and write permissions.</source>
        <translation>Kann das Unterverzeichnis in %1 erstellen. Bitte prüfen Sie, ob Sie Schreib- und Leseberechtigung haben. </translation>
    </message>
</context>
<context>
    <name>LogMessage</name>
    <message>
        <location filename="../source/LogMessage.h" line="27"/>
        <source>Could not find the log dir %1.</source>
        <translation>Log-Verzeichnis %1 nicht gefunden.</translation>
    </message>
    <message>
        <location filename="../source/LogMessage.h" line="38"/>
        <source>Could not open the log %1.</source>
        <translation>Log-Verzeichnis %1 kann icht geöffnet werden.</translation>
    </message>
</context>
<context>
    <name>NetworkTransferManager</name>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="40"/>
        <source>Could not open %1 for writing.
Please make sure that you have write permissions or an existing file is not write protected.</source>
        <translation>%1 kann nicht zum Schreiben geöffnet werden.
Bitte stelle Sie sicher, dass Sie die entsprechenden Zugriffsrechte haben und die Datei nicht schreibgeschützt ist.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="68"/>
        <source>Could not read %1. Please make sure that the path is correct and you have read permissions.</source>
        <translation>%1 kann nicht gelesen werden.
Bitte stelle Sie sicher, dass Sie die entsprechenden Zugriffsrechte haben.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="126"/>
        <source>The connection to the server was refused.</source>
        <translation>Verbindung zum Server wurde abgelehnt.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="128"/>
        <source>The remote host seems to be closed.</source>
        <translation>Der Remote-Host scheint geschlossen zu sein.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="130"/>
        <source>Host could not be found.</source>
        <translation>Host nicht gefunden.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="132"/>
        <source>The connection timed out.</source>
        <translation>Verbindung hatte einen Time-Out</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="134"/>
        <source>The operation was canceled.</source>
        <translation>Vorgang wurde abgebrochen.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="136"/>
        <source>SSL handshake failed.</source>
        <translation>SSL-handshake fehlgeschlagen.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="138"/>
        <source>A temporary network failure occurred.</source>
        <translation>Ein vorübergehender Netzwerkfehler ist aufgetreten.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="140"/>
        <source>The network session failed.</source>
        <translation>Die Netzwerksitzung ist fehlgeschlagen.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="142"/>
        <source>Background request was not allowed</source>
        <translation>Hintergrundanfrage war nicht erlaubt</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="144"/>
        <source>Too many redirects.</source>
        <translation>Zuviele Redirects (Umleitungen)</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="146"/>
        <source>An insecure redirect occurred.</source>
        <translation>Unsicherer Redirect, Abbruch.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="148"/>
        <source>An unknown network error occurred.</source>
        <translation>Unbekannter Netzwerkfehler.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="150"/>
        <source>The proxy connection was refused.</source>
        <translation>Proxy-Verbindung wurde zurückgewiesen.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="152"/>
        <source>The proxy connection was closed.</source>
        <translation>Proxy-Verbindung wurde geschlossen.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="154"/>
        <source>The proxy was not found.</source>
        <translation>Der Proxy-Server wurde nicht gefunden </translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="156"/>
        <source>The proxy connection timed out.</source>
        <translation>Die Verbindung zum Proxy-Server hatte einen Time-Out.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="158"/>
        <source>The proxy server requires authentication.</source>
        <translation>Der Proxy-Server fordert eine Authentifizierung.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="160"/>
        <source>An unknown proxy server error occurred.</source>
        <translation>Unbekannter Fehler beim Proxy-Server </translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="162"/>
        <source>The access to the desired content on the server was denied.</source>
        <translation>Zugang zu den gewünschten Daten verweigert.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="164"/>
        <source>The operation on the server was not permitted.</source>
        <translation>Der Vorgang war auf dem Server nicht erlaubt.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="166"/>
        <source>The content could not be found on the server.</source>
        <translation>daten nicht auf dem Server vorhanden.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="168"/>
        <source>The login data was not correct.</source>
        <translation>Login war nicht korrekt.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="170"/>
        <source>The content could not be resend.</source>
        <translation>DerInhalt konnte nicht gesendet werden.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="172"/>
        <source>A content conflict occurred.</source>
        <translation>Datenkonflikt</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="174"/>
        <source>The content on the server is gone.</source>
        <translation>Inhalt auf dem Server nicht mehr vorhanden.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="176"/>
        <source>An unknown content error occurred.</source>
        <translation>Unbekannter Fehler.</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="178"/>
        <source>The protocol is unknown.</source>
        <translation>Unbekanntes Protokoll</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="180"/>
        <source>An invalid protocol operation occurred.</source>
        <translation>Unvollständige Protokoll-Operation</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="182"/>
        <source>A protocol failure occurred.</source>
        <translation>Protokoll-Fehler</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="184"/>
        <source>An internal server error occurred.</source>
        <translation>Interner Server-Fehler</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="186"/>
        <source>The desired operation is not implemented.</source>
        <translation>Funktion nicht verfügbar</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="188"/>
        <source>The service is unavailable.</source>
        <translation>Service nicht verfügbar</translation>
    </message>
    <message>
        <location filename="../source/NetworkTransferManager.cpp" line="190"/>
        <source>An unknown server error occurred.</source>
        <translation>Unbekannter Server-Fehler</translation>
    </message>
</context>
<context>
    <name>XFile</name>
    <message>
        <location filename="../source/FileHelper.h" line="55"/>
        <source>No error occurred.</source>
        <translation>Kein Fehler.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="56"/>
        <source>An error occurred when reading from the file.</source>
        <translation>Ein Fehler wärend der Dateilesevorgang ist aufgetreten.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="57"/>
        <source>An error occurred when writing to the file.</source>
        <translation>Ein Fehler wärend der Dateischreibvorgang ist aufgetreten.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="58"/>
        <source>A fatal error occurred.</source>
        <translation>Ein schwerwiegender Fehler ist aufgetreten.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="59"/>
        <source>Resource error.</source>
        <translation>Ressourcenfehler.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="60"/>
        <source>The file could not be opened.</source>
        <translation>Die Datei konnte nicht geöffnet werden.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="61"/>
        <source>The operation was aborted.</source>
        <translation>Der vorgang würde abgebrochen.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="62"/>
        <source>A timeout occurred.</source>
        <translation>Eine Zeitüberschreitung ist aufgetreten.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="63"/>
        <source>An unspecified error occurred.</source>
        <translation>Ein nicht spezifizierter Fehler ist aufgetreten.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="64"/>
        <source>The file could not be removed.</source>
        <translation>Die Datei konnte nicht gelöscht werden.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="65"/>
        <source>The position in the file could not be changed.</source>
        <translation>Die Dateiposition konnte nicht geändert werden.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="66"/>
        <source>The file could not be resized.</source>
        <translation>Die Dateigröße konnte nicht geändert werden.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="67"/>
        <source>The file could not be accessed.</source>
        <translation>Die Datei konnte nicht zugegriffen werden.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="68"/>
        <source>The file could not be copied.</source>
        <translation>Die Datei konnte nicht kopiert werden.</translation>
    </message>
    <message>
        <location filename="../source/FileHelper.h" line="69"/>
        <source>Unknown Error.</source>
        <translation>Unbekannter Fehler.</translation>
    </message>
</context>
</TS>
